Gemfile.lock

```diff
@@ -7,61 +7,65 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (6.0.3.4)
-      actionpack (= 6.0.3.4)
+    actioncable (6.1.0)
+      actionpack (= 6.1.0)
+      activesupport (= 6.1.0)
       nio4r (~> 2.0)
       websocket-driver (>= 0.6.1)
-    actionmailbox (6.0.3.4)
-      actionpack (= 6.0.3.4)
-      activejob (= 6.0.3.4)
-      activerecord (= 6.0.3.4)
-      activestorage (= 6.0.3.4)
-      activesupport (= 6.0.3.4)
+    actionmailbox (6.1.0)
+      actionpack (= 6.1.0)
+      activejob (= 6.1.0)
+      activerecord (= 6.1.0)
+      activestorage (= 6.1.0)
+      activesupport (= 6.1.0)
```

Comment:
Think this should be pinned to 6.0.3

Reply:
Actually, probably doesn't matter 👍🏼

Reply:
@MattHall As this is a public gem it's not locked it to `6.0.*` like the Boxt engines or private gems. Was gonna add "_in case it is used in any 6.1.0 projects_" but I doubt it 😄